pywbem/_subscription_manager.py

```diff
@@ -541,6 +557,22 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
             permanent. See :ref:`WBEMSubscriptionManager` for details about
             these ownership types.
 
+          persistence_type (:class:`py:int` or None):
+            The PersistenceType property defines for the WBEM server
```

Comment:
Here you make a jump from the persistence_type parameter to the PersistenceType property without connecting the two. I suggest first defining what the parameter means without mentioning the property name, and then stating that it sets the property.

Reply:
State in the first sentence now that it sets the PersistenceType property